### ISSUE TITLE
Create a sample application with GWT et J2ObjC

### ISSUE THREAD

**Issue by Bathlamos** [Bathlamos/RTDC#6]
*Problem:* The project needs to compile for the web, Android, and iOS. 
*Task:* JSON documents have been synced with the RESTful server / Postgres DB
*Validation:* iOS app that allows users to login

**Comment by Nicolas-M** [comment]
Prof. Guy-Vincent Jourdan has taught the mobile development course only once. He is only familiar with PhoneGap, which he says works well in most situations. However, he noted it depends on your needs. As such, I think we should keep our current solution (GWT & J2ObjC)

**Comment by Nicolas-M** [comment]
Removed notifications as a separate issue should be created for it. Removed reference to Couchbase, and described issue in detail.

**Comment by Nicolas-M** [comment]
Cross-compiling (GWT and J2ObjC) works well from our previous tests. More rigourous tests, such as #30 will be performed.
